### ISSUE TITLE
An example for custom auth ( eg, just user/pass login system) would be nice.

### ISSUE THREAD

**Issue by v3ss0n** [simplabs/ember-simple-auth#1329]
I am trying to wrap my head around implementing custom auth with my own simple Username/pass auth service (only secure cookies , no tokens ). 

Any example would be really nice . Currently it is very very hard to follow.

**Comment by medemi68** [comment]
Join the club. I'm having some issues getting my head around this too. You find packages such as Ember-Simple-Auth, but they don't seem to explain themselves very well. I have the same requirements.

I don't need an authorizer, none of that. I just need something that POSTs to a URL and that's it.

**Comment by marcoow** [comment]
Check out the docs for the [`BaseAuthenticator`](http://ember-simple-auth.com/api/classes/BaseAuthenticator.html) and the implementation of the rather simple [`DeviseAuthenticator`](https://github.com/simplabs/ember-simple-auth/blob/master/addon/authenticators/devise.js). That should give you an idea of the necessary steps  for implementing a custom authenticator.

I agree that a guide on how to write a custom authenticator would be great. /cc @srvance ;)

**Comment by belgoros** [comment]
I'd glad to joins as well. I have an example [app](https://github.com/belgoros/decastore-front) that uses a corporate authentication page, the master branch contains a login button. The thing I'm stuck now is to remove login button and to redirect to the corporate login page in case if the user is nit authenticated (either the first time he/she comes to the app or after logging out). The failed version is on a [separate branch](https://github.com/belgoros/decastore-front/tree/redirect_to_login_gateway). Even after logging out, the user is immediately re-logged in. No way to figure out how to implement it.

**Comment by marcoow** [comment]
cleaning old issues
